netket/nn/activation.py

```diff
@@ -63,19 +78,23 @@ def log_tanh(x):
     return jax.numpy.log(jax.numpy.tanh(x))
 
 
+reim_selu = reim(selu)
+reim_relu = reim(relu)
```

Comment:
add two short docstrings to the two functions

Reply:
I'm not sure how to do this, as all `reim` is a function that returns a function

Reply:
just below
```python
reim_selu=reim(selu)
"""
my beautiful docstring
"""
```